pkg/oc/admin/router/router.go

```diff
@@ -728,11 +779,44 @@ func RunCmdRouter(f kcmdutil.Factory, cmd *cobra.Command, out, errout io.Writer,
 			},
 		},
 	}
+	if cfg.Type == "haproxy-router" && cfg.SyslogSidecar {
+		containers = append(containers, kapi.Container{
+			Name:  "syslog",
+			Image: cfg.ImageTemplate.ExpandOrDie("egress-dns-proxy"),
```

Comment:
This looks funny, but I need an image that includes rsyslog.  Is there a more suitable image we could use here? (Note: BusyBox's syslogd is inadequately flexible; only rsyslog allows reading from a Unix domain socket, for example.)

Reply:
How big is the rsyslog package set?  If it's reasonably small, it should be added to haproxy-router.

Reply:
`du -hc $(rpm -ql rsyslog)` reports 2.1M.